### PR TITLE
Generate Model Documentation

### PR DESCRIPTION
**Pull request by sonlinux**
fix kartoza/LEDET_BIMS#54

sample_documentation

### CODE REVIEW

File: testingdocs.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+|scientific name |scientific_name |varchar(100) | | | |Blank | |
+|author |author |varchar(100) | | | |Blank | |
+
+Options
+```
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## survey-locationsite relationship(bims.models.survey.Survey_sites)
```
Comment (dimasciput) [inline testingdocs.md]: I think you need to put proper verbose_name under meta class in all of these models, so it can generate this doc better.

Comment (sonlinux) [reply testingdocs.md]: Alright @dimasciput let me revisite the bims models

Comment (sonlinux) [reply testingdocs.md]: Hi @dimasciput 
Its okay now, with .rst we have all the models correctly named.
```## survey-locationsite relationship(bims.models.survey.Survey_sites)```
Just defines the relationship on the two models.
Thanks

**Comment by timlinux** [comment]
@sonlinux please up your game with your PRs:

* Proper reference to upstream tickets
* Screenshots / sample outputs
* Description of the work you have done
* Updates to README / system documentation with notes on how to generate docs

Thanks!

**Comment by timlinux** [comment]
Why move it to LEDET? It should be in the generic BIMS. Anything generically useful should go in the core kartoza-bims, only skinning stuff etc and things that are only useful to LEDET should go in LEDET.

Please add to this codebase.

**Comment by sonlinux** [comment]
Thanks @timlinux
Its Done

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+LEDET-BIMS documentation
+==============================
+
+## Technical documentation for LEDET-BIMS Project.
+
+## Firstly you need to be in the base project directory where the management
+ command is run from.
+
+```docs/docs/```
```
Comment (ann26) [inline docs/README.md]: command is run from django-bims folder, and do `make shell`

File: docs/docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+# The encoding of source files.
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'docs'
+copyright = u"2018, Alison Mukoma"
```
Comment (ann26) [inline docs/docs/conf.py]: maybe `2018, Kartoza` ?

File: docs/README.md (hunk 2 above)
Comment (sonlinux) [reply docs/README.md]: Thanks @ann26 updated

File: docs/docs/conf.py (hunk 3 above)
Comment (sonlinux) [reply docs/docs/conf.py]: Thanks
Updated

**Comment by ann26** [comment]
LGTM

**Pull request merged**